docs/cli/usage/mock.md

```diff
@@ -100,6 +99,73 @@ query {
+
+In addition to using the context above to resolve CloudFormation template values, the mock environment will populate [lambda runtime environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) in the following way:
+1. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` will be populated using the AWS credentials that the CLI is currently configured to use
+2. `_HANDLER`, `AWS_REGION`, `AWS_LAMBDA_FUNCTION_NAME`, `LAMBDA_TASK_ROOT`, and `LAMBDA_RUNTIME_DIR` will be set based on the function being mocked
+3. Static defaults will be specified for all other runtime environment variables. The full list of static defaults can be found [here](https://github.com/aws-amplify/amplify-cli/blob/af76446d18bf626ca5f91c3ad41081175c959807/packages/amplify-util-mock/src/utils/lambda/populate-lambda-mock-env-vars.ts#L39)
+
+If any of the default environment values do not work for your mocking use case, you can also specify overrides in a `.env` file within the function directory (ie. `<project root>/amplify/backend/function/<function name>/.env`).
+
+### Connecting to a mock @model table
+If you have a function that connects to a table generated by @model you may want to test out this code against the mock table when running the mock API. To do that, create a `.env` file in the function directory and add the following:
+```
+AWS_REGION=us-fake-1
+DDB_ENDPOINT=http://localhost:62224
+AWS_ACCESS_KEY_ID=fake
+AWS_SECRET_ACCESS_KEY=fake
+API_<api name>_<model>TABLE_NAME=<model>Table
+```
+Ensure that you replace `<api name>` with the name of your API and `<model>` with the name of your model. The environment variable name should also be all caps. For example, if you have an API named "FlightStats" and a model defined as `type Airplane @model {...}` then then last line of the `.env` file should be:
+```
+API_FLIGHTSTATS_AIRPLANETABLE_NAME=AirplaneTable
+```
+
+With the above `.env` configuration, you can then configure the DynamoDB call within the function as follows:
+```javascript
+const ddb = new aws.DynamoDB.DocumentClient({
+    endpoint: process.env.DDB_ENDPOINT,
+});
+const result = await ddb.put({
+    TableName: process.env.API_FLIGHTSTATS_AIRPLANETABLE_NAME,
+    Item: {
+        id: '1234567890',
+        name: 'F-22',
+        description: 'Cool fighter jet',
+    },
+}).promise();
+```
+
+When running in the cloud, all of these environment variables will have valid values with the exception of `DDB_ENDPOINT` which will be undefined. In this case the DynamoDB client will use the default endpoint. When running locally, these environment variables will be populated using the local .env file which specifies the fake values required to connect to the local database. Also ensure that you run `amplify mock api` before running `amplify mock function` so that the local DynamoDB endpoint is active.
```

Comment:
```suggestion
When running in the cloud, all of these environment variables will have valid values except `DDB_ENDPOINT` which will be `undefined`. In this case the DynamoDB client will use the default endpoint. 

When running locally, these environment variables will be populated using the local .env file which specifies the fake values required to connect to the local database. 
```

"Also ensure that you run `amplify mock api` before running `amplify mock function` so that the local DynamoDB endpoint is active." should probably be worked in higher in the flow. This might be easier if we convert it to a "step-by-step" guide-like content.

Reply:
Converted this flow into numbered steps